AntennaTracker/make.inc

```diff
@@ -33,3 +33,5 @@ LIBRARIES += AP_RSSI
 LIBRARIES += AP_RPM
 LIBRARIES += AP_Frsky_Telem
 LIBRARIES += AP_Beacon
+LIBRARIES += AP_Relay
+LIBRARIES += AP_ServoRelayEvents
```

Comment:
We should put this in the common libraries, instead of having it in all vehicles.